### ISSUE TITLE
[Assistive tech and device support] - Keyboard navigation order isn't meaningful or sequential. (11.08.2)

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#52441]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
Enrollment verification

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Keyboard navigation order isn't meaningful or sequential.

### Issue Details
When enough months of enrollment verifications are present to have pagination, each time you move to another page in the pagination sequence your keyboard focus is set to the H2 heading "Your monthly enrollment verifications." For keyboard-only users, putting focus here requires an extra [tab] stop on every page of the verification history, tabbing through the "What if I notice..." additional info component. For screen reader users, putting focus here also means no information is immediately announced indicating where you are in the pagination (or whether the pagination link took you anywhere).

Quick note on severity: This is flagged under experience standard 11.08, which would normally automatically be a launch-blocking issue. But this particular instance is a pretty minor version of this kind of issue, so for this one I'm downgrading it by a severity level for you to prioritize as appropriate.

### Link, screenshot or steps to recreate
1. On test user 301, navigate to [/enrollment-verifications/](https://staging.va.gov/education/verify-school-enrollment/enrollment-verifications/).
2. Activate a screen reader (recommend VoiceOver on Mac or NVDA on Windows).
3. Press [Tab] until you reach the pagination section. Follow the pagination link for page 2.
4. Notice that keyboard focus is set to the H2 heading "Your monthly enrollment verifications." No information is announced by the screen reader about where you are in the pagination.

### VA.gov Experience Standard
[Category Number 11, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.3_A

### Platform Recommendation
I would copy the behavior of [VA.gov search](https://www.va.gov/search/) and [Find a VA Form](https://www.va.gov/find-forms/). When paginating through both of these, focus is set to the "Showing X - Y of Z" message just above results. For screen reader users, this is the first thing announced after following a pagination link -- which means they get immediate information about where they are relative to where they were.

Note that both search and Find a Form also make their "Showing X - Y of Z" message an H2 heading. I do not think that heading structure is necessary for your product. Just the focus management should be sufficient.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

## Platform directions
- Update "Issue Title"
- Link to  Collaboration Cycle Request epic
- Add the VFS team and product name
- Add your name and practice area under Point of Contact(Practice Area)
- Add the Platform Issue. The Issue column from the [VA.gov Experience Standards](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)
- Add the Issue Details and any links, screenshots or steps to recreate
- Add the [VA.gov Experience Standard](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html) number
- Add assignees: VFS PM
- Add labels:
  - VFS team label
  - VA.gov Experience Standard Issue number (CCIssueXX.YY) where XX is the category number and YY is the issue number, per the VA.gov Experience Standards
  - Practice area labels (508/accessibility, IA, content, design) per the VA.gov Experience Standards
  - 508 defect label, if applicable (508-defect-0, 508-defect-1, 508-defect-2, 508-defect-3, 508-defect-4)
  - Launch-blocking label, if applicable

**Comment by briandeconinck** [comment]
@btruongafs Just want to note that this issue was created today, after testing pagination on the re-staged test user 301. Please let me know if you have any questions! cc @shiragoodman
